src/decorators/relationships.js

```diff
@@ -18,32 +18,15 @@ import { singularizeType } from '../utils'
  * @method relatedToMany
  */
 export function relatedToMany (targetOrModelKlass, property, descriptor) {
-  if (typeof targetOrModelKlass === 'function') {
-    return function (target2, property2, descriptor2) {
-      schema.addRelationship({
-        type: target2.constructor.type,
-        property: property2,
-        dataType: Array
-      })
-
-      return {
-        get () {
-          const { type } = targetOrModelKlass
-          return getRelatedRecords(this, property2, type)
-        }
-      }
-    }
-  } else {
-    schema.addRelationship({
-      type: targetOrModelKlass.constructor.type,
-      property,
-      dataType: Array
-    })
-
-    return {
-      get () {
-        return getRelatedRecords(this, property)
-      }
```

Comment:
Actual change

Reply:
this is good, maybe we should make this a major version change. this feels breaking.

Reply:
Yep this is definitely a breaking change since we also return promises from findOne/findAll now too